packages/cli/src/lib/commands/history-util.ts

```diff
@@ -64,36 +62,37 @@ export function getNextDeviceEvents(table: Table, items: DeviceActivity[], optio
 	}
 }
 
-export async function writeDeviceEventsTable(
+export const writeDeviceEventsTable = async (
 		command: SmartThingsCommandInterface,
 		data:  PaginatedList<DeviceActivity>,
-		options?: Partial<DeviceActivityOptions>): Promise<void> {
-
+		options?: Partial<DeviceActivityOptions>): Promise<void> => {
 	const opts = { includeName: false, utcTimeFormat: false, ...options }
 	const head = options && options.includeName ?
 		['Time', 'Device Name', 'Component', 'Capability', 'Attribute', 'Value'] :
 		['Time', 'Component', 'Capability', 'Attribute', 'Value']
 
 	if (data.items) {
-		let table = command.tableGenerator.newOutputTable({ isList: true, head })
+		const table = command.tableGenerator.newOutputTable({ isList: true, head })
 		getNextDeviceEvents(table, sortEvents(data.items), opts)
 		process.stdout.write(table.toString())
 
-		let more = 'y'
-		while (more.toLowerCase() !== 'n' && data.hasNext()) {
-			more = (await inquirer.prompt({
-				type: 'input',
+		while (data.hasNext()) {
+			const more = (await inquirer.prompt({
+				type: 'confirm',
 				name: 'more',
-				message: 'Fetch more history records ([y]/n)?',
-			})).more
-
-			if (more.toLowerCase() !== 'n') {
-				table = command.tableGenerator.newOutputTable({ isList: true, head })
-				await data.next()
-				if (data.items.length) {
-					getNextDeviceEvents(table, sortEvents(data.items), opts)
-					process.stdout.write(table.toString())
-				}
+				message: 'Fetch more history records?',
+				default: true,
+			})).more as boolean
+
+			if (!more) {
+				break
+			}
+
+			const table = command.tableGenerator.newOutputTable({ isList: true, head })
```

Comment:
I didn't know you could re-define a `const` in a more restricted context like this.  Neat.  But could certainly get confusing in a less straightforward situation than this one.